contents/pacman.oscmeta

```diff
@@ -0,0 +1,19 @@
+{
+    "information": {
+        "name": "Pacman",
+        "author": "A.D.Price/D.Ashley/P.Briot",
+        "category": "emulators",
+        "peripherals": ["Wii Remote"],
+        "version": "auto"
+    },
+    "source": {
+        "type": "url",
+        "format": "zip",
+        "location": "https://wiibrew.org/w/images/a/ae/Wii_pacman1_04_20160712.zip"
+    },
+    "treatments": [
+        {
+            "treatment": "contents.move", "arguments": ["apps/", "apps/pacman/"]
+        }
```

Comment:
this is problematic on the server, because it practically tries to move apps into itself.
to fix this you can rename the apps directory from apps to something else and only then move it to the right location.